utils/subscription_feed.ts

```diff
@@ -34,41 +45,74 @@ export const updateSubscriptionStatus = (
   context: Context,
   logPrefix: string,
   version: number,
-  delPartitionKey: string,
-  delKey: string,
-  insPartitionKey: string,
-  insKey: string
+  deleteEntity: SubscriptionFeedEntitySelector,
+  deleteOtherEntities: ReadonlyArray<SubscriptionFeedEntitySelector>,
+  insertEntity: SubscriptionFeedEntitySelector,
+  doesntInsertIfDeleted: boolean
```

Comment:
ditto

Reply:
Fixed in [0a20b29](https://github.com/pagopa/io-functions-app/pull/189/commits/0a20b298929d320f91a74b0302eb9aefb54ccf7e)